spec/spec.md

```diff
@@ -906,13 +897,11 @@ Get all objects of a given schema type:
 ```json
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://identity.foundation/dwn/json-schemas/records-write.json",
+  "$id": "https://identity.foundation/dwn/json-schemas/records-write-unidentified.json",
   "type": "object",
   "additionalProperties": false,
   "required": [
-    "authorization",
-    "descriptor",
-    "recordId"
```

Comment:
I think we need to keep all 3 in here.

Reply:
@andorsk yeah this is just an implementation specific helper-schema that is used when creating a message that has not yet been signed by the signer. But a complete valid RecordsWrite message has to have all of those properties for it to be valid.